### DISCUSSION TITLE
Is that possible to force reorder displays (multiple screens) ?

### DISCUSSION THREAD

**Discussion by Gat0zor** [awawa-dev/HyperHDR#1022] [General]
I recently switch to HyperHDR cause Hyperion was laggy on my W11 system when I was minimizing or restoring windows on my desktop.
In cause was OpenGL whitch is not used into HyperHDR and seems lightest.

But when I select multiple screen it shows in order of screens not in order of what is my desktop with Nvidia or Windows preferencies.
Is that possible to reorder displays ?
Thanks a lot for your help

![firefox_3beh2EZJRn](https://github.com/user-attachments/assets/bfb8f1c6-34a6-4793-89c8-c6b1c666ff1a)

![firefox_IXQZ7tNmhu](https://github.com/user-attachments/assets/4d4b1be3-e783-45df-aaa0-caad778a233c)

**Comment by awawa-dev** [comment]
Hi
Yes, it's a good idea to have the option to change the order of the screens.
Already implemented and merged: https://github.com/awawa-dev/HyperHDR/pull/1023

**Comment by Gat0zor** [comment]
Oh thanks a lot for your reactive feature added ! It will be nice ! 👍
